aws/hhvm1/lambdas/check_if_repos_changed.py

```diff
@@ -16,6 +16,7 @@ def lambda_handler(event, context=None):
     if (
       'success' in vr.get('PublishBinaryPackages', {}) or
       'success' in vr.get('PublishSourceTarball', {}) or
+      'success' in vr.get('BuildAndPublishMacOS', {}) or
```

Comment:
this means we now also run the update indices + invalidate cloudfront steps after Mac stuff is published, I think that's correct...?

Reply:
yep, that's a good thing.

Should probably have this triggered by a cloudwatch rule watching the S3 bucket instead...